lib/CLI/Types.hs

```diff
@@ -13,6 +13,12 @@ import Coffer.Path (Path, EntryPath)
 import Data.Set (Set)
 import Data.List.NonEmpty ( NonEmpty )
 
+data Options = Options
+  { _backend :: Text 
+  , _someCommand :: SomeCommand
```

Comment:
Please use the same field naming convention as used in other data types (`oBackend`, `oSomeCommand`)